src/api/status/public/js/post/index.js

```diff
@@ -0,0 +1,19 @@
+const totalPostsEl = document.getElementById('totalPosts');
+
+const apiHost = new URL(location).origin.replace(/status/, '');
```

Comment:
I don't think this is necessary as origin will only return back `https://dev.api.telescope.cdot.systems/`